### PR TITLE
Add window position flags

### LINKED ISSUE #325: Add --x/--y options defining start position of app
I am unable to set the start position for the app.
I use a frameless app which also makes it unmovable. (this is good in my case)
however the app defailts to center (electron default). 

I think this is due to: 
x: mainWindowState.x,
y: mainWindowState.y,
in main.js.
If I change it to x: 0, y: 0, this has no effect.

Any ideas on how I can fix this?

Thank you,
Ivo

### PR DESCRIPTION
**Pull request by mdh34**
Add `--x` and `--y` flags to set default window position

Fixes #325

### CODE REVIEW

File: src/options/optionsMain.js [hunk 1]
```diff
@@ -33,6 +33,8 @@ export default function (inpOptions) {
     minHeight: inpOptions.minHeight,
     maxWidth: inpOptions.maxWidth,
     maxHeight: inpOptions.maxHeight,
+    x: inpOptions.x,
```
Comment (ronjouch) [inline src/options/optionsMain.js]: @mdh34 thanks for the surprise-PR 🙂! So, what happens if I don't pass `--x`/`--y` here? x gets `undefined`, right? Have you confirmed the `new BrowserWindow({` later call in `mainWindow.js` is later on happy with that?

Comment (mdh34) [reply src/options/optionsMain.js]: @ronjouch upon testing the behaviour is normal without any arguments - no console errors either :)

Comment (mdh34) [reply src/options/optionsMain.js]: @ronjouch setting a default might not work that well either because of different resolution screens?

Comment (ronjouch) [reply src/options/optionsMain.js]: @mdh34 yeah, I was not thinking of setting a default, I was thinking of only adding `x`/`y` to `inpOptions` when we have them from `inpOptions`. Else let's not set it, it's more explicit that we're letting Electron BrowserWindow do its thing with default values:

```javascript
if (inpOptions.x) {options.x = inpOptions.x;}
if (inpOptions.y) {options.y = inpOptions.y;}
```

Comment (mdh34) [reply src/options/optionsMain.js]: Ah yes sounds like a good idea :)

**Comment by ronjouch** [comment]
@mdh34 thanks! Merry end-of-year-something!

**Pull request merged**